backend/src/main/java/controllers/PostController.java

```diff
@@ -256,7 +250,8 @@ private String getPostLikes(String id) {
      */
     private String getPostComments(String id) {
         String[] comments = postManager.getPostComments(id);
-        StringBuilder formattedComments = new StringBuilder("Comments:\n\n");
+        int numComments = comments.length;
+        StringBuilder formattedComments = new StringBuilder("Comments (").append(numComments).append("):\n\n");
```

Comment:
could we just call comments.length on this line? Is there a reason we need to save numComments?